lib/phoenix/test/channel_test.ex

```diff
@@ -239,9 +239,12 @@ defmodule Phoenix.ChannelTest do
   """
   defmacro connect(handler, params) do
     if endpoint = Module.get_attribute(__CALLER__.module, :endpoint) do
+      stringified_params =
+        Code.eval_quoted(params) |> elem(0) |> stringify_params |> Macro.escape
```

Comment:
yeah. I hesitated to make it public in ChannelTest. It could be inlined? What do you think?